Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [LOOT/Prelude](https://hosted.weblate.org/projects/loot/prelude/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/loot/prelude/horizontal-auto.svg)

tygyh committed:
Translated using Weblate (Swedish)

Currently translated at 89.0% (57 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/sv/

Kyo-70 committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (64 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/pt_BR/

pStyl3 commented:
@tygyh
The translation `Denna plugin är en **{0}*, men den innehåller FormID som inte matchar denna typ av Mästare. Att använda denna plugin skulle ha oförutsägbara konsekvenser för ditt spel och rekommenderas därför inte.` is missing a `*`.

It should be:
`Denna plugin är en **{0}**, men..`

Can you fix the message on weblate?

pStyl3 committed:
Translated using Weblate (Swedish)

Currently translated at 89.0% (57 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/sv/

github-actions[bot] committed:
Import translations from Weblate
